packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -156,7 +156,9 @@ Color titleSolidBackgroundColor(ThemeData theme) {
   return theme.isDarkTheme ? devtoolsGrey[900] : devtoolsGrey[50];
 }
 
-final chartBackgroundColor = ThemedColor(Colors.grey[50], Colors.grey[850]);
+// TODO(kenz): can we access this color from the Theme somehow? This mirrors
```

Comment:
This might be ThemeData's `scaffoldBackgroundColor` field?

Reply:
it is! switched.